spec_integration/facter_to_hash_spec.rb

```diff
@@ -48,12 +48,6 @@
         expect(out).to match(/[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}/)
       end
 
-      it 'returns ipaddress6' do
```

Comment:
would it be possible to run this integration test only on OSes that have ipv6 addresses?

Reply:
Removed the commit and created https://tickets.puppetlabs.com/browse/FACT-3032.